### PR TITLE
2.0 Polls akka-apps refactor

### PR DESCRIPTION
**Pull request by antobinary**
Note, the event for sending a whiteboard shape with the results is currently commented out. Once we have the Whiteboard refactoring done we'll enable that piece

### CODE REVIEW

File: akka-bbb-apps/src/main/scala/org/bigbluebutton/core2/message/handlers/GetCurrentPollReqMsgHdlr.scala [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import org.bigbluebutton.core.models.Polls
+import org.bigbluebutton.core.running.MeetingActor
+import org.bigbluebutton.common2.domain.PollVO
+
+trait GetCurrentPollReqMsgHdlr {
+  this: MeetingActor =>
+
+  val outGW: OutMessageGateway
+
+  def handleGetCurrentPollRespMsg(msg: GetCurrentPollReqMsg): Unit = {
```
Comment (capilkey) [inline akka-bbb-apps/src/main/scala/org/bigbluebutton/core2/message/handlers/GetCurrentPollReqMsgHdlr.scala]: Why is the function called "RespMsg"? Shouldn't it be "ReqMsg"?

File: akka-bbb-apps/src/main/scala/org/bigbluebutton/core/models/Polls.scala [hunk 2]
```diff
@@ -0,0 +1,619 @@
...
+    for {
+      poll <- getPoll(pollId, lm.polls)
+    } yield {
+      hidePollResult(pollId, lm.polls)
+      // outGW.send(new PollHideResultMessage(props.meetingProp.intId, props.recordProp.record, requesterId, pollId))
+      pollId
+    }
+  }
+
+  def handleGetCurrentPollReqMsg(requesterId: String, lm: LiveMeeting): (Boolean, Option[PollVO]) = {
```
Comment (capilkey) [inline akka-bbb-apps/src/main/scala/org/bigbluebutton/core/models/Polls.scala]: Couldn't you just return the Option? Some means it exists and None means it doesn't

File: akka-bbb-apps/src/main/scala/org/bigbluebutton/core2/message/handlers/GetCurrentPollReqMsgHdlr.scala (hunk 1 above)
Comment (antobinary) [inline akka-bbb-apps/src/main/scala/org/bigbluebutton/core2/message/handlers/GetCurrentPollReqMsgHdlr.scala]: Good catch! Fixed now.

File: akka-bbb-apps/src/main/scala/org/bigbluebutton/core/models/Polls.scala (hunk 2 above)
Comment (antobinary) [inline akka-bbb-apps/src/main/scala/org/bigbluebutton/core/models/Polls.scala]: Now returning the Option[PollVO] only

**Pull request merged**